src/gl/metal/vcRenderShaders.cpp

```diff
@@ -3,6 +3,8 @@
 const char *const g_VisualizationFragmentShader = "visualizationFragmentShader";
 const char *const g_VisualizationVertexShader = "visualizationVertexShader";
 
+const char *const g_ViewShedFragmentShader = "flatColorFragmentShader"; // not implemented
+const char *const g_ViewShedVertexShader = "udVertexShader"; // not implemented
```

Comment:
I think you can just say `nullptr` for these?

Reply:
I was just trying to get these to compile for Metal. For this PR it's not an issue if this breaks Metal as we're having other issues there. @DamianMadden thoughts on what to do here?

Reply:
Well this shouldn't cause a problem unless the functions are used